### DISCUSSION TITLE
如何快速搭建自己的博客网站 | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#34] [Announcements]
# 如何快速搭建自己的博客网站 | 连波的闲谈杂鱼

前段时间将我的博客开源后，陆陆续续的收到很多朋友的喜欢，也有朋友在部署或自定义时遇到了各种各样的问题，更收到了快 50 岁的非专业程序员的邮件咨询，很是开心

https://godruoyi.com/posts/how-to-build-your-blog/

**Comment by achuanya** [comment]
兄台的博客滚动起来有种粘稠感..

**Reply by godruoyi** [reply]
谢谢反馈了，多半是引入的 https://github.com/darkroomengineering/lenis 有问题，我用 Arc 没复现，等我换个浏览器试试 ❤️。

**Reply by rebron1900** [reply]
哈，到处都有你。

**Comment by 20000428** [comment]
老师你的图片是存在哪的，感觉你图片加载的好快啊:yellow_heart:

**Reply by godruoyi** [reply]
图片是保存在 R2 的，你可以查看[这里的文档(自定义图片上传部分)](https://godruoyi.com/posts/how-to-build-your-blog/#%E5%B0%86%E8%B5%84%E6%BA%90%E4%B8%8A%E4%BC%A0%E5%88%B0-s3-%E5%8A%A0%E9%80%9F%E8%AE%BF%E9%97%AE)如果你也想将所有文件(js/css/img)保存到 R2 作为 CDN 加速的话。

**Reply by 20000428** [reply]
好的谢谢老师我研究一下，另外提个小小意见感觉每篇文章有个回到顶部按钮和文章导航目录会更好:yellow_heart:
